capstone/src/main/java/com/step902020/capstone/Review.java

```diff
@@ -14,32 +21,99 @@
 
   long timestamp;
 
-  @Field(name="name")
-  String name;
+  int likes;
+
+  String individualName;
+
+  String individualEmail;
 
   @Field(name="text")
   String text;
 
+  List<String> reviewLikers;
+
   /**
-   * Create a new Review
+   * Creates a new Review
+   * @param individualName Name of individual who authored review
    * @param text Text content of review
-   * @param name Review's author
    */
-  public Review(String text, String name) {
+  public Review(String individualName, String individualEmail, String text) {
     this.timestamp = System.currentTimeMillis();
+    this.likes = 0;
+    this.individualName = individualName;
+    this.individualEmail = individualEmail;
     this.text = text;
-    this.name = name;
+    this.reviewLikers = new ArrayList();
```

Comment:
are you planning on displaying the people who like a review or is it just to keep track of whether or not a user has liked a review?

Reply:
Right now, it's to keep track so a user can't like more than once